app/models/choice.rb

```diff
@@ -2,6 +2,8 @@ class Choice < ActiveRecord::Base
   belongs_to(:answer, :inverse_of => :choices, :touch => true)
   belongs_to(:option, :inverse_of => :choices)
 
+  validates :answer, :option, presence: true
```

Comment:
Well, i like having both validations, because i like to easily refer to what is needed on the code and also to have clean messages on the model.errors (using validate and that kind of stuff). But as you said, it's not really needed! ;)
And the specs for that is just to make sure someone doesn't accidentally remove something it was meant to be there... again, this is totally questionable.

But this is just me. Please let me know what do you prefer and i'll rebase the changes here. 

Thanks!